gen_test.py

```diff
@@ -0,0 +1,301 @@
+import json
+import random
+import string
+import sys
+import time
+
+import fast_diff_match_patch as fdmp
+
+DEFAULT_ALPHABET = string.ascii_letters + string.digits + string.whitespace
+
+# speedtest1.txt: aaaabbccbcbbacc
+# speedtest2.txt: cbbccbccbbbcbac
+
+def gen_test_random(size1, size2, letters=DEFAULT_ALPHABET):
+    """Generates two strings of size1 and size2 and writes each of them in a file.
```

Comment:
nit: it doesn't seem to be any file writing in this function, nor in some of the other functions with this docstring

Reply:
Right, the docstring is outdated.